devtools/x/src/test.rs

```diff
@@ -49,6 +49,9 @@ pub fn run(mut args: Args, xctx: XContext) -> Result<()> {
     let config = xctx.config();
 
     let env_vars: &[(&str, &str)] = if args.html_cov_dir.is_some() {
+        if !xctx.installer().install_if_needed("grcov") {
+            std::process::exit(1);
```

Comment:
I'm thinking that a better way to handle exiting the process would be to throw an error (well return one) and have some logic at main that prints the error and exits. This way we only terminate the process from a single place cleanly.